### PR TITLE
Create en.yml

### PR DESCRIPTION
**Pull request by dalley-it**

### CODE REVIEW

**Comment by dalley-it** [comment]
Hi 
the spelling of the title has failed because of the words 'Dalley' and 'PhotoSwipe'.
Is it possible to add these two words to the whitelist?

**Comment by Toflar** [comment]
Can you please consolidate all the changes in one pull request? You can continue to commit on the same branch. A PR is merging one branch into another, not single commits.

File: meta/dalley-it/contao-photoswipe/en.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+en:
+    title: Dalley IT - PhotoSwipe
+    description: >
+        Integration of javacsript library photoswipe. Easily adjust your photoswipe galleries in the contao backend.
```
Comment (xchs) [inline meta/dalley-it/contao-photoswipe/en.yml]: Typos: JavaScript, PhotoSwipe, Contao, back end

Comment (dalley-it) [reply meta/dalley-it/contao-photoswipe/en.yml]: Thank you. 
According to the error.log of travis the Typos are Dalley and PhotoSwipe. However your suggestions seem reasonable to me.

Can anyone please give me a hint how to extend the according whitelist.

Sorry I am new to this.

**Pull request closed without merging**